kong/db/schema/typedefs.lua

```diff
@@ -521,6 +521,23 @@ local function validate_path_with_regexes(path)
   return true
 end
 
+local function validate_regex_or_plain_pattern(pattern)
+  if pattern:sub(1, 1) ~= "~" then
+    return true
+  end
+
+  pattern = pattern:sub(2)
+
+  local _, _, err = ngx.re.find("", pattern, "aj")
```

Comment:
Shouldn't we add `"o"` flag as well? The documentation recommends using `"j"` together with `"o"` 🤔 
https://github.com/openresty/lua-nginx-module?tab=readme-ov-file#ngxrematch

Reply:
Not really helpful here as the pattern is different every time. But it should not harm.